.gitlab-ci.yml

```
@@ -3020,9 +3020,7 @@ docker_build_dogstatsd_amd64:
 docker_trigger_internal_amd64:
   stage: image_deploy
   rules:
-    - <<: *if_version_7
-      when: on_success
-    - <<: *if_deploy_7
+    - <<: *if_deploy_on_tag_7
       when: on_success
```

Nit: this bit is not strictly equivalent to #7113:
```suggestion
      when: manual
```